core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -67,16 +67,17 @@ public static JsonObject getTenantMsg(final Message message) {
      * Gets a JSON object representing the reply to a Tenant API request via the vert.x event bus.
      *
      * @param tenantId The tenant for which the message was processed.
-     * @param result The result to return to the sender of the request.
+     * @param tenantResult The result to return to the sender of the request.
      * @return JsonObject The JSON reply object.
+     * @throws NullPointerException If result is null.
```

Comment:
what about tenant ID?

Reply:
Curently, this should never happen, and if it would, the payload would contain a null value for the tenantId key.

But I agree: since it should not occur currently, I will check it here and throw a NPE if tenantId is null, instead of silently setting a null value. It is cleaner to check it, so :+1: